avocado_vt/plugins/vt_runner.py

```diff
@@ -42,42 +42,59 @@ def params(self):
         return self.__vt_params
 
     def run_test(self):
-        params = self.params
+        status = "ERROR"
         try:
-            messages.start_logging(self.config, self.queue)
+            try:
+                messages.start_logging(self.config, self.queue)
+                self._run_test()
+                status = "PASS"
+            except Exception:
+                exc_info = sys.exc_info()
+                status = exc_info[0].status
+                if status == "ERROR" or status not in teststatus.STATUSES:
+                    raise
```

Comment:
This line is rethrowing the exception. I wonder if this is needed. I have the impression that if you remove the first `try` and move line 58 here, you will have the same behavior. Or maybe not :D just wondering.

Reply:
I can't do that, because there might be error during handling of the test exception. But you are right, this has to be changed. There can be handling only for `TestBaseException`, I will change that. thanks